tests/test_insert_deletes_consolidate.cpp

```diff
@@ -79,59 +75,115 @@ inline void load_aligned_bin_part(const std::string& bin_file, T* data,
 }
 
 std::string get_save_filename(const std::string& save_path,
-                              size_t points_to_skip, size_t points_deleted,
+                              size_t points_to_skip, float points_deleted,
+                              float  points_inserted,
                               size_t last_point_threshold) {
   std::string final_path = save_path;
   if (points_to_skip > 0) {
     final_path += "skip" + std::to_string(points_to_skip) + "-";
   }
-  if (points_deleted > 0) {
-    final_path += "del" + std::to_string(points_deleted) + "-";
+  if (points_deleted > 0 && points_inserted > 0) {
```

Comment:
change to something like:
std::string suffix;
if (points_insered > 0)
{
    suffix += "insert" + std::to_string(points_inserted);
}
if (points_deleted > 0)
{
    if (!suffix.empty())
    {
        suffix += "-";
    }
    suffix += "delete" + std::to_string(points_deleted);
}

final_path += suffix

Reply:
ok